### DISCUSSION TITLE
unable to get access to github copilot even after fixing 2F factor.

### DISCUSSION THREAD

**Discussion by Satabdee2000** [community/community#113802] [GitHub Education]
### Hi everyone,

I could get the access to student account even after fixing the 2F factor problem.

**Comment by samus-aran** [comment]
Hi @Satabdee2000  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by Satabdee2000** [comment]
Please check I have sent the mail to you.I have set the 2f factor but after
that it is written that fix the previous problems.

**Comment by Satabdee2000** [comment]
It's writting about github billing details.what is it?
